### PR TITLE
Split log forwarding into separate topics

### PR DESCRIPTION
**Pull request by mburke5678**

### CODE REVIEW

File: _topic_map.yml [hunk 1]
```diff
@@ -1190,9 +1190,12 @@ Topics:
 - Name: Viewing cluster logs using Kibana
   File: cluster-logging-kibana-interface
   Distros: openshift-dedicated
-- Name: Forwarding logs to third party systems
+- Name: Forwarding logs using the Log Forwarding API
   File: cluster-logging-external
   Distros: openshift-enterprise,openshift-webscale,openshift-origin
+- Name: Forwarding logs using a ConfigMap
```
Comment (sichvoge) [inline _topic_map.yml:1196]: Seeing it up here makes me not like it anymore :D Maybe we should keep "Forwarding logs to third party systems" for now and describe the ConfigMap approach here, as it is whats fully supported today, and then leave "Forwarding logs using the Log Forwarding API" as a separate chapter as is. 

Also, the "to third party systems" should come before the log forwarding API.

WDYT?

File: logging/cluster-logging-external-configmap.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+
+You can configure cluster logging to send logs to destinations outside of your {product-title} cluster instead of the default Elasticsearch log store using a ConfigMap:
+
+* *Sending logs using the Fluentd forward protocol*. You can create a Configmap to use the xref:#cluster-logging-collector-fluentd_cluster-logging-external-configmap[Fluentd *forward* protocol] to securely send logs to an external logging aggregator that accepts the Fluent *forward* protocol.
+* *Sending logs using syslog*. You can create a Configmap to use the xref:#cluster-logging-collector-syslog_cluster-logging-external-configmap[syslog protocol] to send logs to an external syslog (RFC 3164) server.
+
+[NOTE]
+====
+Forwarding logs using a ConfigMap will be deprecated when the xref:../logging/cluster-logging-external.adoc#cluster-logging-external[Log Forwarding API] reaches General Availability (GA) status, and will be removed in a future release.
```
Comment (sichvoge) [inline logging/cluster-logging-external-configmap.adoc:16]: s/when/once?

**Review by sichvoge — COMMENTED** [review]
Should we add a bit more details to the log forwarding API and that it creates a layer on top of fluentd to make it easier to configure forwarding logs to third party systems? I recently had a few people that asked if Fluentd and Log Forwarding API are two different things and I had to explain that they do not differ since the API is just a CRD that the CLO takes and generates the corresponding Fluentd config.

Do we also say that it is Tech Preview and that we fully support forwarding logs described in the other section we should link?

File: logging/cluster-logging-external-configmap.adoc (hunk 2 above)
Comment (mburke5678) [reply logging/cluster-logging-external-configmap.adoc:16]: The IBM Style Guide recommends using _once_ to mean _one time_.

File: _topic_map.yml (hunk 1 above)
Comment (mburke5678) [reply _topic_map.yml:1196]:
```
Forwarding logs to Third-Party System
Forwarding logs using the Log Forwarding API
```
If the first topic tells folks how to send logs to external systems, what does the Log Forward API do? <smiley-face>

Comment (mburke5678) [reply _topic_map.yml:1196]: _ the API is just a CRD that the CLO takes and generates the corresponding Fluentd config._
I did not know this. (Is this true when the API goes GA? We use the Fluentd config map to send logs to Kafka, syslog, etc?)

Comment (mburke5678) [reply _topic_map.yml:1196]: _When you use the Log Forwarding API, the Cluster Logging Operator creates or modifies a Fluentd ConfigMap to send logs using the Fluentd *forward* protocol. You are responsible to configure the protocol on the external log aggregator. _

**Comment by mburke5678** [comment]
@sichvoge Any final thoughts on this? I would like to get this PR merge ahead of other changes to these files. 

I changed the wording around deprecating the legacy configmap methods because of the additional text we added that the LF API uses a fluentd configmap. I thought it odd that in one place we say configmaps to forward logs will be deprecated when the LF API is GA, but the LF API uses configmaps.

_Manually configuring a ConfigMap to forward logs will be deprecated when the Log Forwarding API reaches General Availability (GA) status and will be removed in a future release._

**Pull request closed without merging**